samples/mqtt/websocket_connect/README_custom_auth.md

```diff
@@ -0,0 +1,95 @@
+# Websocket Connect with Custom Authentication
+
+[**Return to main sample list**](../../README.md)
+
+This sample makes an MQTT connection and connects through a Custom Authorizer.
+On startup, the device connects to the server and then disconnects.
+This sample is for reference on connecting using a Custom Authorizer.
+Using a Custom Authorizer allows you to perform your own authorization using an AWS Lambda function.
+See Custom Authorizer for more information.
+You will need to setup your Custom Authorizer so that the lambda function returns a policy document.
+See this page on the documentation for more details and example return result.
```

Comment:
`this page on the documentation` should be link, like in this readme: https://github.com/aws/aws-iot-device-sdk-java-v2/tree/main/samples/CustomAuthorizerConnect